### ISSUE TITLE
Serverless fails to install on Ubuntu 17.04

### ISSUE THREAD

**Issue by staleks** [serverless/serverless#3493]
Hi, 

I've tried to install Serverless framework on "new, fresh" installment of Ubuntu 17.04.

I've installed NodeJS using Ubuntu official repository. Version that I was offered is NodeJS v4.7.2.
After that following instructions, I've installed npm, again same from official Ubuntu official repository.
Version installed v3.5.2.

Finally, when I started to install `Serverless` Framework, I've got the following error:

astoisavljevic@aleksandar-virtualbox:~$ sudo npm install -g serverless
/usr/local/bin/serverless -> /usr/local/lib/node_modules/serverless/bin/serverless
/usr/local/bin/slss -> /usr/local/lib/node_modules/serverless/bin/serverless
/usr/local/bin/sls -> /usr/local/lib/node_modules/serverless/bin/serverless

> serverless@1.11.0 postinstall /usr/local/lib/node_modules/serverless
> node ./scripts/postinstall.js

sh: 1: node: not found
npm ERR! Linux 4.10.0-19-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "serverless"
npm ERR! node v4.7.2
npm ERR! npm  v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! serverless@1.11.0 postinstall: `node ./scripts/postinstall.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the serverless@1.11.0 postinstall script 'node ./scripts/postinstall.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the serverless package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./scripts/postinstall.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs serverless
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls serverless
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/astoisavljevic/npm-debug.log

As requested, I'm attaching npm-debug.log file to this ticket.

Thank You

[npm-debug.txt](https://github.com/serverless/serverless/files/940919/npm-debug.txt)

**Comment by pmuens** [comment]
Thanks for opening @staleks !

Hmm. Serverless should work with Node > v4.4...

Just to make sure that it has nothing todo with the Node version. Could you please share the exact output you get when you run `which node` and `node --version`? Thanks!

**Comment by staleks** [comment]
Sure thing.

Here is the output:

astoisavljevic@aleksandar-virtualbox:~$ sudo which nodejs
/usr/bin/nodejs
astoisavljevic@aleksandar-virtualbox:~$ sudo nodejs --version
v4.7.2
astoisavljevic@aleksandar-virtualbox:~$

**Comment by pmuens** [comment]
```
astoisavljevic@aleksandar-virtualbox:~$ sudo which nodejs
/usr/bin/nodejs
```

That seems to be the problem. The Serverless bin expects `node` and not `nodejs` AFAIK. Is there any change you can create an alias so that `/usr/bin/node` points to `/usr/bin/nodejs`.

**Comment by HyperBrain** [comment]
Is there any reason why you need to execute node in a root context via sudo? Is the node executable accessible in your user context too? Just try `which nodejs`. Additionally the executable should at least be available under the name "node" and not only "nodejs".

**Comment by pmuens** [comment]
BTW. On my machine `which node` returns `/usr/local/bin/node`

**Comment by staleks** [comment]
Ok, by the information you gave me, I've looked for other resources/blogs how to provide nodejs alternatives.

Solution that worked for me"

1. Create symlink node to nodejs
` sudo ln -s `which nodejs` /usr/bin/node`

2. then I was able to start npm serverless installation, but without `sudo` command, I was getting an error:
`astoisavljevic@aleksandar-virtualbox:~$ npm install -g serverless
npm WARN checkPermissions Missing write access to /usr/local/lib`

3. Finally with `sudo` it worked
`astoisavljevic@aleksandar-virtualbox:~$ sudo npm install -g serverless`
